canonical-web-and-design/jaas-dashboard
# Added analytics pings for opening and saving charm config.

## Done

In order to determine if this feature is being used this branch adds a couple surface level analytics pings.
- when the config is opened.
- when a config value is saved.

## QA

- Pull code.
- `yarn build`
- Using your server of choice `cd build && http-server .`
- Open the devtools to the network pane.
- Find a charm and open the config, it should send an analytics ping.
- Save a config value and it should send an analytics ping.
